jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileCrawler.kt

```diff
@@ -83,7 +83,7 @@ abstract class CodeWhispererFileCrawler : FileCrawler {
         val openedFiles = runReadAction {
             FileEditorManager.getInstance(psiFile.project).openFiles.toList().filter {
                 it.name != psiFile.virtualFile.name &&
-                    isSameDialect(psiFile.virtualFile.extension) &&
```

Comment:
it will always return true since we are provdiing the target file(where users trigger codewhisperer) instead of crossfile candidates